api/README.md

```diff
@@ -4,7 +4,7 @@
 
 | Env Variable                  | Required | Default Value | Description                                                                                                                                                                                                                                                                                                                                                                       |
 | ----------------------------- | -------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| LOG_LEVEL                     | no       | INFO          | Sets the lowest level to the pino logger that is printed to `STDOUT` with varying levels ranging from `trace` to `fatal`.                                                                                                                                                                                                                                                         |
```

Comment:
This env var still exists in the project, but it is unused. Remove it

Reply:
Removed